webrecorder/browsertrix
# Improve org routing & performance

Changes:
- Refactor org routing to use [url-pattern](https://github.com/snd/url-pattern) instead of [path-parser](https://github.com/troch/path-parser). path-parser doesn't support optional segments or parameters, so it was getting unwieldy adding a new named route for each parameter when everything routes to the `btrix-org` component anyway.
- Move org settings to separate component file to better organize.
- Use orgs in `/me-with-orgs` response instead of separate API call to decrease UI load time.
- Fix "Org Settings" visible for non-org owners when switching between orgs

### Manual testing
Pre-req: Run `yarn` to install new dependencies
1. Log in as a user that belongs to multiple orgs
2. Switch between orgs. Verify that "Org Settings" link doesn't show for non-org owner
3. Navigate around app. Verify links work as expected.
